instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/__init__.py

```diff
@@ -355,10 +355,23 @@ def _instrument(self, **kwargs):
 
         is_sql_commentor_enabled = kwargs.pop("is_sql_commentor_enabled", None)
 
+        middleware_position = kwargs.pop("middleware_position", 0)
```

Comment:
Can you add an `OTEL_PYTHON_DJANGO_MIDDLEWARE_POSITION` env so that auto instrumentation people who use `opentelemetry-instrument` can also use this capability?